make/autoconf/toolchain.m4

```diff
@@ -879,7 +884,13 @@ AC_DEFUN_ONCE([TOOLCHAIN_SETUP_BUILD_COMPILERS],
 
       # On windows, the assember is "ml.exe". We currently don't need this so
       # do not require.
-      UTIL_LOOKUP_PROGS(BUILD_AS, ml, [$VS_PATH])
+      if test "x$OPENJDK_TARGET_CPU_BITS" = "x64"; then
```

Comment:
If this is for the BUILD_AS, shouldn't we be checking OPENJDK_BUILD_CPU_BITS?

Reply:
Yes, technically you are correct. I will update. It is a bit of an academic exercise though, since this is not being used. But still, it's good to have for consistency.